### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by rica213**
Use JavaScript classes instead of objects and arrays.
Create class methods to add and remove books.
Maintain all of the original functionality of the application
Add CSS styles to the application to make it match the wireframe

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -10,78 +10,84 @@ class Book {
...
 
 booksContainer.addEventListener('click', (e) => {
-  books.splice(e.target.id, 1);
+  bookList.books.splice(e.target.id, 1);
   e.target.parentElement.remove();
 
-  localStorage.setItem('books', JSON.stringify(books));
-  initInput();
+  localStorage.setItem('books', JSON.stringify(bookList.books));
+  bookList.initInput();
```
Comment (jasonfelice) [inline js/script.js:92]: - [ ] Kindly refactor this bit of code and declare a `remove` method inside the Books class instead.

_Something like this..._

```
  remove(id) {
    this.splice(id, 1);
    localStorage.setItem('books', JSON.stringify(this.books));
  }
```
And you can call it like this: `bookList.remove(id)`

Good Luck 😄

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @rica213, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Good Design✔️
Professional Documentation✔️
No Linter errors✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/script.js (hunk 1 above)
Comment (chepkok3) [reply js/script.js:92]: @jasonfelice Thank you for the suggested solution.

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @rica213,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**